.github/workflows/npm-release.yml

```
@@ -0,0 +1,67 @@
+          releaseVersion: ${{inputs.releaseVersion}}
+
+  
+  build-image:  
+    runs-on: ubuntu-latest
+    needs: release
+    steps:
+      - uses: it-at-m/.github/.github/actions/action-build-image@main
+        with:
+          path: "${{  inputs.app-path }}"
+          image-name: "${{  inputs.app-path }}"
+          artifact-name:  ${{ needs.release.outputs.ARTIFACT_NAME }}
```

yamllint (1.35.1)

[error] too many spaces after colon

(colons)